VMBackup/main/hostsnapshotter.py

```diff
@@ -76,6 +79,14 @@ def snapshotall(self, paras, freezer, g_fsfreeze_on, taskId):
                         temp_dict[CommonVariables.key] = flag
                         temp_dict[CommonVariables.value] = paras.wellKnownSettingFlags[flag]
                         settings.append(temp_dict)
+                '''
+                HostBased changes, hence commenting out
+                if(paras.isVMADEEnabled == True):
+                    settings.append({CommonVariables.key:CommonVariables.isOsDiskADEEncrypted, CommonVariables.value:self.isOsDiskADEEncrypted})
+                    settings.append({CommonVariables.key:CommonVariables.areDataDisksADEEncrypted, CommonVariables.value:self.areDataDisksADEEncrypted})
+                    meta_data.append({CommonVariables.key:CommonVariables.encryptionDetails, CommonVariables.value:self.encryptionDetails})
+                '''
```

Comment:
we can skip adding this for now and include it in next PR to keep the clean set of changes.